package.json

```diff
@@ -1,15 +1,16 @@
 {
   "devDependencies": {
+    "ajv": "6.1.1",
     "eslint": "4.13.1",
     "eslint-config-prettier": "2.9.0",
+    "husky": "^0.14.3",
```

Comment:
Whenever they release version 0.15 (release candidate right now) the way the script hooks are configured will change (https://github.com/typicode/husky#upgrading-from-014). Should we just explicitly stick to 0.14.3 ?

Reply:
That is a good catch. I read about it but forgot. Yeah, let's lock this down to 0.14.3 for now and then when 0.15 is released we can use the new syntax. I actually prefer the new syntax since it doesn't clutter up the `scripts` section.